src/components/AddOn/core/AddOnStore/AddOnStore.tsx

```diff
@@ -76,13 +72,13 @@ function addOnStore(): JSX.Element {
     store.dispatch({ type: 'UPDATE_P_TARGETS', payload: links });
   };
   // /* istanbul ignore next */
-  const pluginModified = (): void => {
-    return getInstalledPlugins();
-    // .then((installedPlugins) => {
-    //   getStorePlugins();
-    //   return installedPlugins;
-    // });
-  };
+  // const pluginModified = (): void => {
```

Comment:
I think this is not a valid function, as you said `pluginInstallStatus ` is no longer supported. I think we can completely remove this func?

Reply:
Yeah its not a valid function. I will remove the commented code and update the file.

Reply:
Hello, I have updated the file.